README.md

```diff
@@ -102,6 +103,14 @@ The following changes are now available in production:
 
 ### self-assessment-accounts-api
 
+The following changes are now available in production:
+
+* Version 1.0 has been deprecated, see [sandbox changelog](https://github.com/hmrc/income-tax-mtd-changelog/wiki#9th-january-2023)
```

Comment:
I don't think V1 was ever actually deployed to production. It's disabled in the [config](https://github.com/hmrc/app-config-production/blob/main/self-assessment-accounts-api.yaml#L44).